demo/site/src/common/components/Typography.tsx

```diff
@@ -194,14 +194,14 @@ const variantToElementMap: Record<TypographyVariant, "h1" | "h2" | "h3" | "h4" |
 export const Typography = styled.div.attrs<{
     as?: unknown;
     variant?: TypographyVariant;
-    bottomSpacing?: boolean;
+    $bottomSpacing?: boolean;
```

Comment:
We recently fixed this in https://github.com/vivid-planet/comet/pull/3083, which is cleaner IMO.

Reply:
Ah nice 👍🏼 
Do you mean you prefer `shouldForwardProp` in general over transient props, or just in this case? 🤔